### DISCUSSION TITLE
Sequential Convex Programming

### DISCUSSION THREAD

**Discussion by CodingTil** [coin-or/Ipopt#535] [Q&A] [answered]
Is it possible to solve non-convex optimization problems (such as [this one](https://www.researchgate.net/publication/335495086_Distributed_Model_Predictive_Pose_Control_of_Multiple_Nonholonomic_Vehicles)) using SCP (Sequential Convex Programming). In the paper, they used an algorithm (algorithm 1) presented in [another paper](https://www.researchgate.net/profile/Bassam-Alrifaee/publication/318280544_Sequential_Convex_Programming_MPC_for_Dynamic_Vehicle_Collision_Avoidance/links/5978b094a6fdcc30bdc31d05/Sequential-Convex-Programming-MPC-for-Dynamic-Vehicle-Collision-Avoidance.pdf).
I don't know much about this library and SCP, sorry.

**Comment by svigerske** [comment, marked as answer]
Ipopt implements an interior point algorithm only. There are no plans to add other, significantly different, algorithms.
